### PR TITLE
updates golang sdk to v0.20.0

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

File: router/xgress_edge_tunnel/fabric.go [hunk 1]
```diff
@@ -48,6 +49,10 @@ import (
 	"time"
 )
```
Comment (plorenz) [inline router/xgress_edge_tunnel/fabric.go:51]: I feel like we should add this to foundation. Tiny, but generally useful thing

**Review by plorenz — APPROVED** [review]
Looks good. All the pointer values make me a bit nervous, but I'm guessing you checked that they're always set.

File: router/xgress_edge_tunnel/fabric.go (hunk 1 above)
Comment (andrewpmartinez) [reply router/xgress_edge_tunnel/fabric.go:51]: Looks like it is already there in foundation/util/util.go. It is called `Ptr()`

**Comment by andrewpmartinez** [comment]
> Looks good. All the pointer values make me a bit nervous, but I'm guessing you checked that they're always set.

Most of the pointers come from the generated go-swagger rest_mode. Go-swagger uses `nil` for `nullable` and for `required`/`optional` (to defeat zero values like 0, false). The clients validate the payloads well before they hit our code, so if something is required that is missing, an error is generated higher up in the go-swagger client.

**Pull request merged**